package/src/common/update-html-dependencies.ts

```diff
@@ -487,7 +487,7 @@ async function updatePdfJs(
     "pdf",
     "pdfjs",
   );
-  moveSync(from, to, { overwrite: true });
+  copySync(from, to, { overwrite: true });
```

Comment:
copySync is necessary because from/to may be on different partitions (temp fs to a disk). Moving/renaming doesn't work under those conditions.